### DISCUSSION TITLE
problem updating photoprism from 1.0.9 to 1.0.11

### DISCUSSION THREAD

**Discussion by RAMPropertyManagementPtyLtd** [photoprism/photoprism#2937] [Q&A]
Any help would be appreciated, 
I am running photoprism as an application through Truenas scale.
It works extremely well and i have been updating it as new versions are appearing.
only problem i am having is trying to update it from version 1.0.9 to 1.0.11
It updates then deploys and activates but soon after goes back to deploying and stays there forever until i downgrade back to version 1.0.9 and then it deploys and activates fine on the older version.
Am i missing some steps in the update process or is anyone else experiencing the same issue?
regards
Dale

**Comment by lastzero** [comment]
These version numbers don't look familiar to us. Seems best to ask the TrueNAS team.

**Reply by RAMPropertyManagementPtyLtd** [reply]
ok will do thanks for the prompt reply
